### PR TITLE
docs: Fixes some code formatting

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: doc/book/container/factories.md [hunk 1]
```diff
@@ -37,18 +37,18 @@ order to seed the `Application` instance:
...
-      ],
-      // An array of middleware to register after registration of the
-      // routing middleware:
-      'post_routing' => [
-      ],
+```php
+'middleware_pipeline' => [
+  // An array of middleware to register prior to registration of the
+  // routing middleware:
+  'pre_routing' => [
```
Comment (weierophinney) [inline doc/book/container/factories.md]: No; the original is correct and follows our coding standards (4 space indentation).

File: doc/book/router/uri-generation.md [hunk 2]
```diff
@@ -13,41 +13,41 @@ the name:
 - If you call `route()` with no HTTP methods, the name is the literal path with
   no changes.
 
-  ```php
-  $app->route('/foo', $middleware); // "foo"
-  ```
```
Comment (weierophinney) [inline doc/book/router/uri-generation.md]: No. This was correctly indented; RTFD.org simply doesn't recognized fenced code blocks associated with bullet points. While we're initially publishing on RTFD.org, that is not the final location; the final location will use bookdown to render the documentation, and bookdown renders this correctly.

**Comment by samsonasik** [comment]
Ok, closing ;)

**Comment by weierophinney** [comment]
Closing; every single case I looked at in this PR either does formatting that contra-indicates our own coding standards, or changes the semantics for specific elements.

**Comment by samsonasik** [comment]
I already closed it

**Pull request closed without merging**